### PR TITLE
load sas files from the dai server

### PR DESCRIPTION
**Pull request by mtanco**

### CODE REVIEW

File: data/load_sas7bdat.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		if not isinstance(dai_file_path, str):
+			raise ValueError("Variable: 'dai_file_path' should be <str>")
+		if not isinstance(file_names, list):
+			raise ValueError("Column: 'file_names' should be <list>")
+
+		# TODO: add checs that files exist
+		data_sets = {}
+
+		for f in file_names:
+		    full_path_to_file = dai_file_path + f
```
Comment (goldentom42) [inline data/load_sas7bdat.py]: best here is to use `os.path.join(dai_file_path, f)`, this works on any operating system.
you need to `import os` first

File: data/load_sas7bdat.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		    with SAS7BDAT(full_path_to_file, skip_header=False) as reader:
```
Comment (goldentom42) [inline data/load_sas7bdat.py]: to check if a file exists, just use : `os.path.exists(full_path_to_file)`

**Comment by nkpng2k** [comment]
https://github.com/h2oai/driverlessai-recipes/pull/75/files bit of housekeeping only... but I was thinking it might be good to have separated directory for data ingest from 3rd party sources, e.g sas

**Comment by mtanco** [comment]
@nkpng2k Do you think that counts here where we are not pull from a sas server or anything, we are just teaching DAI how to read local files that are the .sas7_bdat file type? So they are starting a special folder on the DAI server already (wherever end user put them).

**Pull request merged**